.github/workflows/build.yml

```diff
@@ -11,13 +11,27 @@ jobs:
           wget https://svn.reactos.org/storage/vperevertkin/rosbe-ci.tar.zst
           mkdir ${{github.workspace}}/rosbe
           tar -I zstd -xvf rosbe-ci.tar.zst --directory ${{github.workspace}}/rosbe
+      - name: Install other packages
+        run: sudo apt install ccache
```

Comment:
Do the build containers get reused?
I think the main benefit of ccache comes from building ROS a second time when a previous build has been cached -- so if we get a fresh container each build, I would imagine it to not be useful.